src/query/functions/aggregation/take.go

```diff
@@ -68,9 +68,10 @@ func NewTakeOp(
 
 // takeOp stores required properties for take ops
 type takeOp struct {
-	params   NodeParams
-	opType   string
-	takeFunc takeFunc
+	params      NodeParams
+	opType      string
+	opTypeBytes []byte
```

Comment:
question: why have both opType and opTypeBytes?

Reply:
Just looking to avoid having to cast it to and from `string` where necessary